app/controllers/certificates_controller.rb

```diff
@@ -0,0 +1,9 @@
+class CertificatesController < ApplicationController
+  def show
+    @certificate = Certificate.find(params[:id])
+    send_data CertificatePDF.new(@certificate).render,
+              filename: "#{@certificate.id}.pdf",
```

Comment:
Use one level of indentation for parameters following the first line of a multi-line method call.